packages/components/src/components/post-header/post-header.scss

```diff
@@ -205,6 +205,7 @@
     height: calc(100vh - var(--header-height));
     display: flex;
     flex-direction: column;
+    overflow-y: auto;
```

Comment:
This also introduces the possibility to scroll the first level navigation when the second one is open:

![image](https://github.com/user-attachments/assets/dc77a724-ea13-45c8-acda-69a0c3360ee8)

When the second level navigation is open, the first level navigation should not be scrollable.